Ginger/GingerCoreNET/WorkSpaceLib/WorkSpace.cs

```diff
@@ -462,7 +462,8 @@ public bool OpenSolution(string solutionFolder, string encryptionKey = null)
                 }
 
                 Reporter.ToLog(eLogLevel.INFO, "Loading Solution- Loading Solution file xml into object");
-                Solution solution = SolutionOperations.LoadSolution(solutionFile, true, encryptionKey);
+                string txt = File.ReadAllText(solutionFile);
```

Comment:
Why we need a solution deserialization twice if we are maintaining old value in temp location

Reply:
Ok , removing this dependency